lib/std/build/LibExeObjStep.zig

```diff
@@ -2060,3 +2060,24 @@ test "addPackage" {
     const dupe = exe.packages.items[0];
     try std.testing.expectEqualStrings(pkg_top.name, dupe.name);
 }
+
+test "addVcpkgPaths" {
```

Comment:
```suggestion
test "addVcpkgPaths" {
    if (builtin.os.tag == .wasi) return error.SkipZigTest;

```
Sorry, missed the skip for wasi.